.github/workflows/_check_clang_static_analyzer_latest_sdk.yml

```diff
@@ -0,0 +1,58 @@
+name: Run Clang Static Analyzer (master SDK)
```

Comment:
How would you feel about putting this job inside the already existing checker ?
And run one job or the other depending on a parameter
We would have two jobs : one with the docker, the other one with the clone